build.sh

```diff
@@ -16,9 +16,9 @@ function setGlobals() {
   TOOLKIT_WORK_DIR="${ARTIFACTS_DIR}/toolkit"
   PACKAGES_DIR="${TOOLKIT_WORK_DIR}/packages"
   TEMPLATES_DIR="${SCRIPT_PATH}/models/containerized_model"
+  BASE_IMAGES_DIR="${SCRIPT_PATH}/models/containerized_model/base_images"
   NOTEBOOK_DIR="${SCRIPT_PATH}/notebooks"
   TUTORIALS_DIR="${SCRIPT_PATH}/tutorials"
-  BUILD_REPORT="${ARTIFACTS_DIR}/buildReport.txt"
```

Comment:
This was a txt version of of `buildReport.json` that wasn't being used - not needed